package.json

```diff
@@ -42,6 +42,8 @@
         "typescript": "4.3.2"
     },
     "dependencies": {
+        "@google-cloud/storage": "^5.19.2",
+        "babel-eslint": "^10.1.0",
```

Comment:
I think it was probably a mistake to add these to the root package.json, yeah?  Make sure to remove them if so.

Reply:
These should probably get removed, yeah?